### PR TITLE
Add battery information accessors

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
## Change list

Depends on 
https://github.com/appium/appium-uiautomator2-driver/pull/164
https://github.com/appium/appium-xcuitest-driver/pull/661
 
## Types of changes

- [ ] No changes in production code.
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Comment by mykola-mokhnach** [comment]
@KazuCocoa FYI

File: src/main/java/io/appium/java_client/android/AndroidBatteryInfo.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        super(input);
+    }
+
+    /**
+     * @return Battery status value.
+     */
+    public BatteryStatus getStatus() {
+        final int status = ((Long) getInput().get("status")).intValue();
+        switch (status) {
+            case 2:
```
Comment (vmaxim) [inline src/main/java/io/appium/java_client/android/AndroidBatteryInfo.java]: I think it would be better to move these magic numbers to enum.

Comment (mykola-mokhnach) [reply src/main/java/io/appium/java_client/android/AndroidBatteryInfo.java]: these are used only once in this particular place. Creating a separate enum for them would be an overkill.

**Comment by mykola-mokhnach** [comment]
Resolved

**Comment by TikhomirovSergey** [comment]
@mykola-mokhnach Going to run Android tests today.
@SrinivasanTarget What about iOS part?

**Comment by mykola-mokhnach** [comment]
@TikhomirovSergey Android is still not merged. There are problems with server publishing. I'll add a comment when it's done

**Comment by TikhomirovSergey** [comment]
@mykola-mokhnach Ok. Maybe there is sense to publish 6.0.0 without this change? How about 6.0.1?

**Comment by mykola-mokhnach** [comment]
The Pr for uia2 driver has been merged, so this PR can be merged as well.

**Comment by SrinivasanTarget** [comment]
@TikhomirovSergey We are good to release 6.0.0 with this now :)

**Pull request merged**